test/core/list/sort.wren

```diff
@@ -0,0 +1,19 @@
+class A {
+    value { _value }
+
+    toString { _value.toString }
+
+    construct new(value) {
+        _value = value
+    }
+
+    <(other) { _value < other.value }
+
+    >=(other) { _value >= other.value }
+}
+
+System.print([3, 1, 5, 9, 7].sort()) // expect: [1, 3, 5, 7, 9]
+
+System.print([A.new(3), A.new(1), A.new(5), A.new(9), A.new(7)].sort()) // expect: [1, 3, 5, 7, 9]
+
+System.print([A.new(3), A.new(1), A.new(5), A.new(9), A.new(7)].sort{|n| n.value }) // expect: [1, 3, 5, 7, 9]
```

Comment:
Maybe try sorting by something different here to verify that the list isn't being sorted by the `<(other)` and `>(other)` impementations. Could try `-n.value` maybe?